CONTRIBUTING.md

```diff
@@ -0,0 +1,856 @@
+# Bug reports
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Do check to make sure your bug hasn't already been filed. Please give the following information, where possible:
+
+1. The version of Mithril you're using, whether it's the dev version or [the version on npm](http://npm.im/mithril.js). The version on npm may not have all the latest bug fixes, so your bug might very well be fixed in the dev version.
```

Comment:
And whether they're using the minified version or not. (Just in case they run into the trailing comma issue with an old version of IE, since it'll (likely) work in the minified version, but not the expanded version)